src/generic/graphicc.cpp

```diff
@@ -2237,7 +2250,7 @@ wxCairoContext::wxCairoContext( wxGraphicsRenderer* renderer, const wxMemoryDC&
 #endif
 
 #ifdef __WXMAC__
-    CGContextRef cgcontext = (CGContextRef)dc.GetWindow()->MacGetCGContextRef();
+    CGContextRef cgcontext = (CGContextRef)dc.GetSelectedBitmap().GetHBITMAP();
```

Comment:
what is the reason for this change ? IIRC this bitmap-context does not know about its content scaling

Reply:
Thanks, this works too (but saved images are the same, so maybe the bitmap-context does know about it).